pkg/odo/genericclioptions/localprovider.go

```diff
@@ -24,38 +19,5 @@ func GetValidEnvInfo(cmdline cmdline.Cmdline) (*envinfo.EnvSpecificInfo, error)
 		return nil, err
 	}
 
-	// Now we check to see if we can skip gathering the information.
-	// Return if we can skip gathering configuration information
-	configIsNeeded, err := cmdline.CheckIfConfigurationNeeded()
-	if err != nil {
-		return nil, err
-	}
-	if !configIsNeeded {
-		return envInfo, nil
-	}
-
-	// Check to see if the environment file exists
-	if !envInfo.Exists() {
-		exitMessage := `The current directory does not represent an odo component. 
-
-To get started,%s
-  * Open this folder in your favorite IDE and start editing, your changes will be reflected directly on the cluster.
-
-Visit https://odo.dev for more information.`
```

Comment:
With this gone, we solely rely on the existence of devfile.yaml to determine if the directory represents an odo component.
```sh
$ odo delete component                                      
 ✗  no devfile found
```
Can you add a message about next steps when no devfile is present?